Assets/MixedRealityToolkit/_Core/Interfaces/IMixedRealityDevice.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+namespace Microsoft.MixedReality.Toolkit.Internal.Definitions
+{
+    public interface IMixedRealityDevice
```

Comment:
needs doc comments

Reply:
yup

Reply:
Seems to be the same as other interfaces we've made.